runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1247,6 +1247,25 @@ gcParseXXArguments(J9JavaVM *vm)
 		}
 	}
 
+	{
+		IDATA portableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXPORTABLESHAREDCACHE, NULL);
+		IDATA noportableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXNOPORTABLESHAREDCACHE, NULL);
+		if (portableSharedCacheIndex > noportableSharedCacheIndex) {
+			extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible = true;
+		}
+		else if (portableSharedCacheIndex < noportableSharedCacheIndex) {
+			extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible = false;
+		}
+		else {
+			OMRPORT_ACCESS_FROM_J9PORT(vm->portLibrary);
+			if (TRUE == omrsysinfo_is_running_in_container()) {
+				extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible = true;
+			}
+			else {
+				extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible = false;
+			}
+		}
+	}
```

Comment:
I guess if you set `extensions->shouldForceToLowMemoryHeapCeilingShiftIfPossible = true` you should always set 
`extensions->forcedShiftingCompressionAmount = 3` as well (default is 0)